diff --git a/score/mw/com/test/api_idempotency/main_api_idempotency.cpp b/score/mw/com/test/api_idempotency/main_api_idempotency.cpp
index 312ac479..c1f1a844 100644
--- a/score/mw/com/test/api_idempotency/main_api_idempotency.cpp
+++ b/score/mw/com/test/api_idempotency/main_api_idempotency.cpp
@@ -193,9 +193,7 @@ void CheckRepeatedDiscovery(
     }
 }
 
-}  // namespace
-
-static void RunApiIdempotencyTest()
+void RunApiIdempotencyTest()
 {
     const auto instance_specifier_result =
         InstanceSpecifier::Create(std::string{kApiIdempotencyInstanceSpecifierString});
@@ -308,6 +306,7 @@ static void RunApiIdempotencyTest()
     WaitForServiceAvailability(instance_specifier, false);
 }
 
+}  // namespace
 }  // namespace score::mw::com::test
 
 int main(int argc, const char** argv)
