diff --git a/BUILD b/BUILD
index a64d2a9..6beead9 100644
--- a/BUILD
+++ b/BUILD
@@ -55,12 +55,12 @@ copyright_checker(
     name = "copyright",
     srcs = [
         ".github",
+        "BUILD",
+        "MODULE.bazel",
         "quality",
         "score",
         "third_party",
         "tools",
-        "//:BUILD",
-        "//:MODULE.bazel",
     ],
     config = "//third_party/cr_checker:config",
     template = "//third_party/cr_checker:templates",
